kolena/_api/v1/generic.py

```diff
@@ -68,6 +68,7 @@ class Path(str, Enum):
         LOAD_INFERENCES = "/generic/test-run/load-test-sample-inferences"
         UPLOAD_INFERENCES = "/generic/test-run/upload-inferences"
         UPLOAD_TEST_SAMPLE_METRICS = "/generic/test-run/upload-test-sample-metrics"
+        UPLOAD_TEST_SAMPLE_METRICS_THRESHOLDED = "/generic/test-run/upload-test-sample-metrics-thresholded"
```

Comment:
nit: the type is `ThresholdedMetrics`, correct? We should be consistent with how the term is used.
```suggestion
        UPLOAD_TEST_SAMPLE_THRESHOLDED_METRICS = "/generic/test-run/upload-test-sample-thresholded-metrics"
```

Reply:
updated